spec/lib/good_job/lockable_spec.rb

```diff
@@ -199,6 +199,80 @@
     end
   end
 
+  describe '.advisory_record_lock' do
+    it 'results in a locked record' do
+      model_class.advisory_record_lock!(key: "good_jobs-#{execution.active_job_id}")
```

Comment:
@bensheldon, I don't really like adding this prefix here `good_jobs-` and I also don't want to modify the passed key on a class level, because in a use case which you have shown in the issue itself:
https://github.com/bensheldon/good_job/blob/6f0a3fa828e1937184241de8d3d6b9b8d7ce12d2/lib/good_job/active_job_extensions/concurrency.rb#L33-L42
we are passing key with the included prefix

Reply:
I agree that makes sense. I think it's fine to have `key` operate slightly differently at the class level than the instance level; in my mind this interface is still exploratory.

- Class level `key`: no prefix, generic.
- Instance level `key`: prefixed, instance-specific.

You can use `execution.lockable_key` in this test rather than manually prefixing the string.